README.md

```diff
@@ -354,90 +354,132 @@ settings that machine might have that could differ from the machine used for wri
+
+Then you can build the docker images by using a [docker-compose](https://docs.docker.com/compose/install/) command which will build the images, create an admin user in development or test environment, compiling the Frontend Assets and run the containers automatically for you:
 
 ```
 $  docker-compose up
 ```
-Once the container is up and ready, go to **http://localhost:8080/** to see your server ready to use
+This command starts the following services:
+
+| Name        | Description                                                                | Port(s)            | Environment(s)                                   |
+|-------------|----------------------------------------------------------------------------|--------------------|--------------------------------------------------|
+| nodejs          | A nodejs container to compil css                                                | n/a               | all      |
+| database          | A MySQL 5.7 database server                                                | 3306               | all (prefer using a managed service in prod) |
+| php         | The OpenCfp project with PHP, PHP-FPM 7.4, Composer and sensitive configs     | n/a                | all                                              |
+| nginx       | The HTTP server for the OpenCfp project (NGINX)                               | 8080               | all                                              |
+
+<details>
+  <summary>To see the status of the containers, run:</summary>
+
+  ```bash
+  docker-compose ps
+  ```
+</details>
+
+<details>
+  <summary>To run any command in the app container you can use the docker-compose exec command:</summary>
+
+  ```bash
+  docker-compose exec <container name> <command>
+  docker-compose exec php sh # To enter the container directly, you will be placed at the root of the project
+  $ docker-compose exec php bin/console cache:clear  # To execute a cache:clear with the php console
+  ```
+</details>
+
+<details>
+  <summary>To see the container's logs, run:</summary>
+
+  ```bash
+  docker-compose logs        # display the logs of all containers
+  docker-compose logs -f     # same but follow the logs
+  docker-compose logs -f <container_name> # follow the logs for one container
+  ```
+</details>
 
-If an administrator already exists, an error message will be displayed to you. You can ignore it if you've used docker-compose before.
+<details>
+  <summary>To rebuild the containers:</summary>
 
-Sometimes you only want to rebuild the images to create your container. To do that use
+  ```bash
+  docker-compose build        # Rebuild all the containers
+  ```
+</details>
+
+Once the containers are up, you will have to add an administrator for the website.
+To do that you only have to execute 
 
 ```sh
-$ docker-compose build
+
+$ docker-compose exec php bin/console user:create --first_name="CHANGE HERE" --last_name="CHANGE HERE" --email="CHANGE HERE" --password="CHANGE HERE" --admin
```

Comment:
`docker-compose exec php bin/console user:create --first_name="CHANGE_HERE" --last_name="CHANGE_HERE" --email="CHANGE_HERE" --password="CHANGE_HERE" --admin`

Reply:
Is not changed.